datasource/utils.go

```diff
@@ -4,6 +4,61 @@ package datasource
 
 import "strconv"
 
+var DivisionalAreas = [3]string{"Arts and Humanities", "Social Sciences", "Science & Engineering & Applied Science"}
+
+func getGenEdInfo(intAttributes []interface{}) []string {
+
+	// Type conversion, since we're dealing with an interface.
+	var attributes []map[string]interface {}
+	for _, item := range intAttributes {
+		attrMap, ok := item.(map[string]interface {}) 
+		if !ok {
+			panic("An item in courseAttributes is not of type map[string]interface")
+		}
+		attributes = append(attributes, attrMap)
+	}
+
+	areas := []string{}
+	for _, attribute := range attributes {
+		if attribute["crseAttributeDescription"] == "FAS General Education" {
```

Comment:
There are some with description `FAS General Education` and others with description `FAS: General Education`. To avoid confusion I think it's preferable to check the `crseAttribute` code for new courses instead, which is LGE.

Reply:
<discussed in meeting>